sdk/storage/storage-file-share/test/directoryclient.spec.ts

```diff
@@ -196,6 +196,48 @@ describe("DirectoryClient", () => {
     assert.ok(result.fileId!);
     assert.ok(result.fileParentId!);
   });
+  
+  it("create with all parameters configured setting filePermission format", async () => {
+    const getPermissionResp = await shareClient.getPermission(
+      defaultDirCreateResp.filePermissionKey!,
+      {
+        filePermissionFormat: "Binary"
```

Comment:
we might should use "SDDL" format  in some case although it's default value.

Reply:
Added one more case